src/script.h

```diff
@@ -90,7 +91,7 @@ typedef struct scriptFlag {
 
 extern scriptFlag scripts_flags_def[];
 
-void scriptPrepareForRun(scriptRunCtx *r_ctx, client *engine_client, client *caller, const char *funcname);
+int scriptPrepareForRun(scriptRunCtx *r_ctx, client *engine_client, client *caller, const char *funcname, uint64_t function_flags, int ro);
```

Comment:
i explicilty asked Meir to put the flags in script.c and not function.c since i planed on eval to use them, so they're not function flags, they're script flags.
```suggestion
int scriptPrepareForRun(scriptRunCtx *r_ctx, client *engine_client, client *caller, const char *funcname, uint64_t flags, int ro);
```

Reply:
:+1: